mmpose/models/backbones/resnet.py

```diff
@@ -280,7 +280,7 @@ def get_expansion(block, expansion=None):
     1. If ``expansion`` is given, just return it.
     2. If ``block`` has the attribute ``expansion``, then return
        ``block.expansion``.
-    3. Return the default value according the the block type:
+    3. Return the default value according the block type:
```

Comment:
Could you change the `according` to `according to` as well? Thanks~

Reply:
> 

Thanks for your help!